fieldsolver/fs_common.cpp

```diff
@@ -236,53 +277,57 @@ std::array<Real, 3> interpolatePerturbedB(
    xLocal[2] -= 0.5;
 
    if (fabs(xLocal[0]) > 0.5 || fabs(xLocal[1]) > 0.5 || fabs(xLocal[2]) > 0.5) {
-      cerr << __FILE__ << ":" << __LINE__ << ": Coordinate (" << xLocal[0] << "," << xLocal[1] << "," << xLocal[2] << ")  outside of this cell!" << endl;
+      cerr << __FILE__ << ":" << __LINE__ << ": Coordinate (" << xLocal[0] << "," << xLocal[1] << "," << xLocal[2]
+           << ")  outside of this cell!" << endl;
       abort();
    }
 
-   std::array<int, 3> cellIds = {i,j,k};
+   std::array<int, 3> cellIds = {i, j, k};
    std::array<Real, Rec::N_REC_COEFFICIENTS> rc;
-   
-   if(FieldTracing::fieldTracingParameters.useCache) {
-      #pragma omp critical
+
+   if (FieldTracing::fieldTracingParameters.useCache) {
+#pragma omp critical
       {
          if (reconstructionCoefficientsCache.find(cellIds) == reconstructionCoefficientsCache.end()) {
-            reconstructionCoefficients(
-               perBGrid,
-               dPerBGrid,
-               rc,
-               i,
-               j,
-               k,
-               3 // Reconstruction order of the fields after Balsara 2009, 2 used for general B, but 3 used here to allow for cache reuse, see interpolatePerturbedJ below
+            reconstructionCoefficients(perBGrid,
+                                       dPerBGrid,
+                                       rc,
+                                       i,
+                                       j,
+                                       k,
+                                       3 // Reconstruction order of the fields after Balsara 2009, 2 used for general B,
+                                         // but 3 used here to allow for cache reuse, see interpolatePerturbedJ below
             );
```

Comment:
So I would prefer what's red in this spot, but TBH I'll get used to the style and/or run clang-format when needed as long as we agree on it and get going. :)

Reply:
I agree, the green one looks pretty nasty.